components/release/base/kustomization.yaml

```
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  # Core manifests
+  - https://github.com/konflux-ci/konflux-ci/konflux-ci/release/core?ref=df1ab193b2b9f6224a4c09a5a2ee1bc0f3a4ae57
+  # CertManager manifests
+  - https://github.com/konflux-ci/konflux-ci/konflux-ci/release/certmanager?ref=df1ab193b2b9f6224a4c09a5a2ee1bc0f3a4ae57
```

```suggestion
  - https://github.com/konflux-ci/konflux-ci/konflux-ci/release?ref=4496810db939d3e3379a8acad49ebd5aba0571c8
```
Refer to the root kustomization file which already contains all the resources